static/docs/get-started/add-files.md

```diff
@@ -36,28 +36,30 @@ committed to Git to track versions of your file:
     $ git commit -m "add source data to DVC"
 ```
 
-To modify or replace a data file that is under DVC control you may need to run
-`dvc unprotect` or `dvc remove` first (check the
-[Update Tracked File](/doc/user-guide/update-tracked-file) guide). Refer to
-[Data and Model Files Versioning](/doc/use-cases/data-and-model-files-versioning)
-and `dvc add` for more information. Use `dvc move` to rename or move a data file
-that is under DVC control.
```

Comment:
I moved the paragraph above to after the `<details>` section, into 2 paragraphs which I think makes more sense. See the new version lines 58+